docs/CHANGELOG.rst

```diff
@@ -4,14 +4,26 @@ Change Log
 
 All notable changes to this project are documented in this file.
 
+
 ==========
 Unreleased
 ==========
 
+Changed
+-------
+* Removed obsolete utilities superseded by resolwe-runtime-utils
+
+Fixed
+-----
+* Included all source files and supplementary package data in sdist
+
 Added
 -----
 * "flow_collection: sample" to processes
```

Comment:
I would rather do it tomorrow when we'll prepare a new release and as part of preparing the release, we will also clean up/prettify the CHANGELOG.

The reason I'm against it is that I don't want to fix unrelated things in commits. Making a separate commit for this change seems overkill if the release is so close.